thread-fields.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Thread fields
+description: Extend and customise Plain's data model.
+---
+
+To be able to keep track of additional information related to a support request, you can configure additional custom fields you want to store on a thread.
+
+To configure the schema for thread fields go to **Settings** → **Thread fields**
+
+Thread fields can either be:
+
+- A boolean (true or false)
+- Text
+- A dropdown (single select from a predetermined list)
+
+Thread fields can also be nested and be conditional on answers from other fields.
+
+Fields can also be marked as required. This is useful when there is some information you want to collect on every support request before it is marked as done.
+
+Thread fields can also be **auto-filled via AI**. To enable this behaviour in the settings for any thread field enable "Autofill via AI".
```

Comment:
could be worth mentioning that the thread field's description is used as context for the AI, so advise is to add that if autofill AI is enabled